scripts/extract-latest-release-notes.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * This extracts the latest (non-"Unreleased") notes from a CHANGELOG.md and saves them to `./.tmp/release-notes.md`
+ *
+ * Usage (from project root):
+ * - node scripts/extract-latest-release-notes.js [PATH TO CHANGELOG]
+ * Examples:
+ * - node scripts/extract-latest-release-notes.js ./packages/CHANGELOG.md
+ * - node scripts/extract-latest-release-notes.js ./experimental/CHANGELOG.md
+ */
+
+const fs = require('fs');
+
+const changelog = fs.readFileSync(process.argv[2]).toString();
+const firstReleaseNoteEntryExp = /(## \d+.\d+.\d\n*(.*\n)*?)(?=## )/m;
```

Comment:
nit: The regex could be clarified a little bit:
- use `^` for start of lines
- use `s` flag so that `.` can match newlines, then matching all the lines between the h2's is a simple `.*?`
- escape the dots in the version

```suggestion
const firstReleaseNoteEntryExp = /^## \d+\.\d+\.\d\n.*?(?=^## )/ms;
```

Reply:
Yes, that's better, thanks :+1: